apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -230,6 +365,73 @@ class SbbolFintechApi extends SbbolRequestApi {
     get apiPrefix () {
         return '/fintech/api'
     }
+
+    /**
+     * Get extended client info request, posted with `getClientInfoRequest`
+     *
+     * @return {Promise<{error: any}|{ClientInfo: responseClientInfo}>}
+     */
+    async getClientInfo () {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getClientInfoRequestPath,
+        })
+
+        try {
+            const jsonData = JSON.parse(data)
+            const error = get(jsonData, 'error')
+
+            if (error) return logger.error({ msg: 'getClientInfo from SBBOL error', error })
+
+            if (statusCode === 200) {
+                return { ClientInfo: jsonData }
+            } else {
+                return { error: jsonData, statusCode }
+            }
+        } catch (error) {
+            return logger.error({ msg: 'getClientInfo from SBBOL error', error })
+        }
+    }
+
+    get getClientInfoRequestPath () {
+        return `${this.apiPrefix}/v1/client-info`
+    }
+
+    /**
+     * Get transactions request, posted with `getStatementTransactionsRequest`
+     * @typedef StatementTransactions
+     * @property {accountNumber} SbbolAccount number
+     * @property {statementDate} Date - optional
+     * @property {page} page - optional
+     * @return {Promise<{error: any}|{data: StatementTransactions}>}
+     */
+    async getStatementTransactions (accountNumber, statementDate, page = 1) {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getStatementTransactionsRequestPath,
+            body: {
+                accountNumber,
+                statementDate,
+                page,
+            },
+        })
+
+        try {
+            const jsonData = JSON.parse(data)
+
+            if (statusCode === 200) {
+                return { data: jsonData }
+            } else {
+                return { error: jsonData, statusCode }
+            }
+        } catch (error) {
+            return logger.error({ msg: 'getStatementTransactions from SBBOL error', error })
+        }
+    }
```

Comment:
This piece of code gets repeated over and over in each method.
Maybe to incapsulate is in super-class `SbbolRequestApi` and to handle `200` and `201` response codes in a success condition fork?